### ISSUE TITLE
Missing pages

### ISSUE THREAD

**Issue by bigdawgsfootball** [Averell7/PdfBooklet#14]
This is a great program, thanks so much for providing it.
I have noticed an issue on one of the files I work with. I generally make a booklet where I fit 2 books on the same 8.5"x11" page. On page 4, the bottom booklet misses the actual page 4 and instead uses what would be on the opposite page - this throws off the rest of the booklet. At seemingly random pages, the behavior repeats. The booklet at the top is fine throughout the document.
![pdfbookletError](https://user-images.githubusercontent.com/8102723/67623190-937a7a80-f7f0-11e9-9241-63df00f75517.png)

Do you know of any way to work around this bug? I can provide the entire file I use if needed.

**Comment by bigdawgsfootball** [comment]
This is against version 3.1.0 and 2.3.3

**Comment by Averell7** [comment]
> I can provide the entire file I use if needed.

Yes, please, because I have never seen this and I am unable to reproduce on 3.1.0

**Comment by bigdawgsfootball** [comment]
> Yes, please, because I have never seen this and I am unable to reproduce on 3.1.0

Sent to personal email.
